benchmarks/run_all_benchmarks.sh

```
@@ -178,13 +178,34 @@ echo ""
+labels = set()
+for p in pathlib.Path(sys.argv[1]).glob("*.json"):
+    try:
+        labels.add(json.loads(p.read_text())["system"]["machine"])
+    except Exception:
+        pass
+# Refuse to guess: deleting by the wrong label would delete another machine s
+# results. Zero or several labels means do nothing and say so.
+print(labels.pop() if len(labels) == 1 else "")
+' "${RUN_DIR}" 2>/dev/null)
+    if [ -n "${MACHINE}" ]; then
+      OLD_COUNT=$(ls -1 benchmarks/results/${MACHINE}_*.json 2>/dev/null | wc -l)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)